modules/ecs6-class/line.js

```diff
@@ -2,10 +2,17 @@ const Point = require("./point");
 
 class Line {
     constructor({ point1 = new Point(), point2 = new Point(), n = undefined, slope = undefined }) {  
-        this.point1 = point1;
-        this.point2 = point2;
-        this.slope = slope;
-        this.n = n;
+       if((point1 instanceof Point) )      
+          if((point2 instanceof Point)){
+            this.point1 = point1;
+            this.point2 = point2;
+            this.slope = slope;
+            this.n = n;
```

Comment:
if n is not undefined but is also not a number
if slope is not undefined but also not a number???

Reply:
i fixed it .